agenta-cli/agenta/cli/helper.py

```diff
@@ -1,11 +1,36 @@
-from pathlib import Path
-from typing import Any, List, MutableMapping
-import click
+import sys
 import toml
+import click
+import questionary
+from pathlib import Path
 from agenta.client import client
+from typing import Any, List, MutableMapping
 from agenta.client.api_models import AppVariant
 
 
+def get_api_key():
+    agenta_dir = Path.home() / ".agenta"
+    credentials_file = agenta_dir / "config.toml"
```

Comment:
let's simply call it credentials. without an extension

Reply:
Oh, check the review comments above. @mmabrouk asked me to change it to this from credentials. 
See it [here](https://github.com/Agenta-AI/agenta/pull/787#discussion_r1369828107)

Reply:
@aakrem:
1. not having an extension does not work with toml files
2. credentials is specific, we'll probably have other data there, why limit ourselves